src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/MvcViewFeaturesLoggerExtensions.cs

```diff
@@ -77,18 +84,23 @@ static MvcViewFeaturesLoggerExtensions()
             _viewResultExecuting = LoggerMessage.Define<string>(
                 LogLevel.Information,
                 1,
-                "Executing ViewResult, running view at path {Path}.");
+                "Executing ViewResult, running view {ViewName}.");
 
-            _viewFound = LoggerMessage.Define<string>(
+            _viewFound = LoggerMessage.Define<string, double>(
                 LogLevel.Debug,
                 2,
-                "The view '{ViewName}' was found.");
+                "The view path '{FilePath}' was found in {ElapsedMilliseconds}ms.");
```

Comment:
Take a look a line 62. Either this should be `{ViewFilePath}` or that one should be `{FilePath}`. I'm ok with either choice just be consistent 👍

Reply:
Oops. Didn't mean to do that. Thanks